### ISSUE TITLE
Request/Question, Unable to pass ava arguments via targets array

### ISSUE THREAD

**Issue by subash-canapathy** [sindresorhus/grunt-ava#4]
Tried to pass ava test runner flags as additional array argument, and as one string. This doesnt work when grunt is ran
`ava: {
      target: ["src/tests/ava/*.js", "--match='@bvt*'"]
 }`

`ava: {
      target: ["src/tests/ava/*.js --match='@bvt*'"]
 }`

Output:
target: ["src/tests/ava/_.js", "--match='@bvt_'"]  => All tests are ran, the second arg in array is ignored ?
target: ["src/tests/ava/_.js --match='@bvt_'"] =>  1 exception

```
  ✖ Couldn't find any files to test
```

**Comment by sindresorhus** [comment]
That unfortunately not supported yet.

**Comment by subash-canapathy** [comment]
so no options can be given apart from the fileset itself ?

**Comment by sindresorhus** [comment]
No, not until we land https://github.com/sindresorhus/grunt-ava/pull/3. You can put it in package.json though.
